xds/internal/test/xds_server_serving_mode_test.go

```diff
@@ -158,15 +161,15 @@ func (s) TestServerSideXDS_ServingModeChanges(t *testing.T) {
 	}
 
 	// Create a ClientConn to the first listener and make a successful RPCs.
-	cc1, err := grpc.Dial(lis1.Addr().String(), grpc.WithTransportCredentials(insecure.NewCredentials()))
+	cc1, err := grpc.Dial(lis1.Addr().String(), grpc.WithTransportCredentials(insecure.NewCredentials()), grpc.WithResolvers(resolver))
```

Comment:
Why didn't this need `WithResolver()` before?

Reply:
You are right. We don't need it here. In this test, the server uses xDS, but the client does not. Removed it.